tests/test_internal_grammar.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
this test can go into tests/test_overrides_parser.py.

Reply:
Yup, just did that in 0c894b3de as mentioned elsewhere :)